### PR TITLE
feat(soft-delete): Add permanent delete document with queue worker and trigger by cron

### PR DESCRIPTION
**Pull request by fajarhikmal214**
## Overview
- feat(soft-delete): Add permanent delete document with queue worker and trigger by cron every mid night

## Related Task
- https://app.clickup.com/t/3dtxf8r

## Evidence
title: feat(soft-delete): Add permanent delete document with queue worker and trigger by cron every mid night
project: SIDEBAR
participants: @fajarhikmal214 @indraprasetya154 @samudra-ajri

### CODE REVIEW

File: src/app/Http/Traits/SignActionDocumentSignatureTrait.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ protected function doSignature($setupConfig, $data, $documentSignatureEsignData)
         ]);
         if ($response->status() != Response::HTTP_OK) {
             $bodyResponse = json_decode($response->body());
-            $this->setPassphraseSessionLog($response, $documentData, SignatureDocumentTypeEnum::UPLOAD_DOCUMENT(), $documentSignatureEsignData);
```
Comment (indraprasetya154) [inline src/app/Http/Traits/SignActionDocumentSignatureTrait.php:100]: Ini kayaknya dari aku aja kang, soalnya udah perlu di up duluan

**Review by indraprasetya154 — APPROVED** [review]

**Review by samudra-ajri — APPROVED** [review]
Awesome! 👍 
Furthermore, I think it will be nice if the overview includes the motive why this feature is developed. This will help the other reviewer to ease to understand. 🚀

**Comment by fajarhikmal214** [comment]
> 

I see. Thank you for your suggestions.

**Comment by fajarhikmal214** [comment]
> Awesome! 👍 Furthermore, I think it will be nice if the overview includes the motive why this feature is developed. This will help the other reviewer to ease to understand. 🚀

I see. Thank you for your suggestions.

**Pull request merged**